sources/application/UI/Base/Component/Field/Field.php

```diff
@@ -354,4 +356,28 @@ public function SetInputType(string $sInputType)
 
 		return $this;
 	}
+
+	/**
+	 * @return string
+	 */
+	public function GetTooltip(): string
+	{
+		return $this->sTooltip;
+	}
+
+	/**
+	 * @param string $sTooltip
+	 *
+	 * @return $this
+	 */
+	public function SetTooltip(string $sTooltip)
+	{
+		$this->sTooltip = $sTooltip;
+		return $this;
+	}
+
+	public function HasTooltip() : bool
+	{
+		return $this->GetTooltip() !== '';
+	}
```

Comment:
```suggestion
	 * @return string
	 * @since 3.1.0
	 */
	public function GetDescription(): string
	{
		return $this->sDescription;
	}

	/**
	 * @param string $sDescription
	 *
	 * @return $this
	 * @since 3.1.0
	 */
	public function SetDescription(string $sDescription)
	{
		$this->sDescription = $sDescription;
		return $this;
	}

	/*
	 * @return bool
	 * @since 3.1.0
	 */
	public function HasDescription(): bool
	{
		return utils::IsNotNullOrEmptyString($this->GetTooltip());
	}
```

_Mind to import the \utils class_